src/main/java/gui/ShotBlock.java

```diff
@@ -28,7 +28,7 @@
     private String description;
 
     // The name of this shotblock
-    @Getter @Setter
+    @Getter
```

Comment:
Because I define setName() lower in the class, as it also does extra stuff now. (update name of the timetableblock)